barretenberg/cpp/src/barretenberg/stdlib/encryption/ecdsa/ecdsa_impl.hpp

```diff
@@ -137,7 +137,6 @@ bool_t<Builder> ecdsa_verify_signature(const stdlib::byte_array<Builder>& hashed
     // Step 8.
     // We reduce result.x() to 2^s, where s is the smallest s.t. 2^s > q. It is cheap in terms of constraints, and
     // avoids possible edge cases
-    // BIGGROUP_AUDITTODO: mutable accessor needed for self_reduce()
```

Comment:
`reduce_mod_target_modulus()` is already a const function, so this todo was not relevant anymore.

Reply:
didnt expect self_reduce to be const! interesting. I guess the idea is that it changes values in such a way that the field value is equivalent?